### PR TITLE
purge: gas settings

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- This aims at retiring the gas settings on the site 
- Remove grant creation for contract_version v0 

Flows tested 

- Tip via Townsquare
- Tips ( send + receive )
- Kudos ( send + receive )
- Bounty 
  - Create
  - Submit Work
  - Basic Payout
  - Advanced Payout
  - Increase Funding
  - Cancel Bounty
- Grants
  - Create
  - Cancel
  - One Time Contribution 
  - Recurring Contirbution

### CODE REVIEW

File: app/assets/v2/js/grants/fund.js [hunk 1]
```diff
@@ -541,7 +526,7 @@ const subscribeToGrant = (transactionHash) => {
           web3.utils.toChecksumAddress(selected_token), // token denomination / address
           web3.utils.toTwosComplement(Number(grant_amount * Math.pow(10, decimals)).toLocaleString('fullwide', {useGrouping: false})),
           web3.utils.toTwosComplement(realPeriodSeconds),
-          web3.utils.toTwosComplement(data.gas_price),
+          web3.utils.toTwosComplement(1),
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/fund.js:529]: TODO: check why subminer fails

File: app/assets/v2/js/grants/fund.js [hunk 2]
```diff
@@ -506,13 +492,13 @@ const subscribeToGrant = (transactionHash) => {
...
         'token_symbol': $('#token_symbol').val(),
-        'gas_price': $('#gas_price').val(),
         'sub_new_approve_tx_id': transactionHash,
         'num_tx_approved': $('#period').val(),
         'network': $('#network').val(),
         'match_direction': $('input[name=match_direction]:checked').val(),
         'is_postive_vote': ($('input[name=match_direction]:checked').val() == '-') ? 0 : 1,
-        'csrfmiddlewaretoken': $("#js-fundGrant input[name='csrfmiddlewaretoken']").val()
+        'csrfmiddlewaretoken': $("#js-fundGrant input[name='csrfmiddlewaretoken']").val(),
+        'gas_price': 1
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/fund.js:501]: TODO: check what to set as gas_price here ?

File: app/assets/v2/js/grants/fund.js (hunk 1 above)
Comment (x5engine) [reply app/assets/v2/js/grants/fund.js]: do you want a fast and expensive gas or a slow and cheap one? or in the middle?

File: app/assets/v2/js/grants/new.js [hunk 3]
```diff
@@ -149,21 +149,6 @@ const init = () => {
           // trusted relayer
           web3.utils.toChecksumAddress(data.trusted_relayer)
         ];
-      } else if ($('#contract_version').val() == 0) {
```
Comment (danlipert) [inline app/assets/v2/js/grants/new.js:152]: Ready to retire the v0 deploy? I think I'm cool with it...

File: app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js [hunk 4]
```diff
@@ -40,41 +37,38 @@ var gas_amount = function(page_url) {
...
-  // only check for updating gas estimates IFF we're on a page where
-  // update_metamask_conf_time_and_cost_estimate() is defined
-  if (typeof update_metamask_conf_time_and_cost_estimate != 'undefined') {
-    setTimeout(function() {
-      updateInlineGasEstimate();
-    }, 500);
-    jQuery('input').change(updateInlineGasEstimate);
-    jQuery('#gasPrice').keyup(update_metamask_conf_time_and_cost_estimate);
-  }
+// jQuery(document).ready(function() {
```
Comment (danlipert) [inline app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js:41]: 如何して

File: app/assets/v2/js/grants/new.js (hunk 3 above)
Comment (thelostone-mc) [reply app/assets/v2/js/grants/new.js:152]: YASS

File: app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js (hunk 4 above)
Comment (thelostone-mc) [reply app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js:41]: Lol i forgot to remove that :P

**Pull request merged**